subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -82,8 +82,8 @@ public void onOutput(OutputEvent event) {
         if (event instanceof ProgressStartEvent) {
             ProgressStartEvent startEvent = (ProgressStartEvent) event;
             // if it has no parent ID, assign this operation as the root operation
-            if (startEvent.getParentId() == null && BUILD_PROGRESS_CATEGORY.equals(startEvent.getCategory())) {
-                rootOperationId = startEvent.getOperationId();
+            if (startEvent.getParentProgressOperationId() == null && BUILD_PROGRESS_CATEGORY.equals(startEvent.getCategory())) {
```

Comment:
Didn't know that we also have a hard-coded category here. We can clean that up with the next PR that maps build operation descriptor to logging type.